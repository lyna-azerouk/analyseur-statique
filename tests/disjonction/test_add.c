{
  int x;
  int y;
  x = rand(10,20);
  y = rand(0,10);
  if(y>5){ x = y - x; }
  else { x = y + x; }
  print_all;
}