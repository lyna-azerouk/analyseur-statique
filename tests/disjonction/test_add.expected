tests/disjonction/test_add.c:8.2-12: [ x in [-14;0] U [10;25], y in [6;10] U [0;5] ]
analysis ended